src/components/Admin/Admin.test.jsx

```diff
@@ -5,10 +5,14 @@ import { mount } from 'enzyme';
 import { MemoryRouter, Link } from 'react-router-dom';
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
+// eslint-disable-next-line no-unused-vars
+import { sendTrackEvent } from '@edx/frontend-platform/analytics';
 
 import EnterpriseDataApiService from '../../data/services/EnterpriseDataApiService';
 import Admin from './index';
 
+jest.mock('@edx/frontend-platform/analytics');
```

Comment:
do we need to make any assertions related to the event being sent in the tests?

Reply:
Sure, we haven't asserted on Segment events in the past in our frontend repos, but I added it to these tests.